proto/cofidectl_plugin/v1alpha1/plugin.proto

```diff
@@ -199,6 +202,7 @@ message ListFederationsResponse {
 
```

Comment:
DestroyFederationRequest should use id

Reply:
It uses proto.federation.v1alpha1.Federation, do we want to replace that with federation_id?

Reply:
Yes - in the Datasource interface I updated all Destroy methods to use an ID.

Reply:
I see!